go.mod

```diff
@@ -35,10 +36,13 @@ require (
 	k8s.io/client-go v0.22.3
 	k8s.io/cluster-bootstrap v0.22.3
 	k8s.io/component-base v0.22.3
+	k8s.io/component-helpers v0.22.3
 	k8s.io/controller-manager v0.22.3
+	k8s.io/klog v1.0.0
```

Comment:
please use k8s.io/klog/v2 instead of k8s.io/lklog

Reply:
done

Reply:
but `k8s.io/klog` is still imported